cpp/src/copying/contiguous_split.cu

```diff
@@ -95,8 +96,11 @@ struct dst_buf_info {
   int dst_offset;     // my offset into the per-partition allocation
   int value_shift;    // amount to shift values down by (for offset buffers)
   int bit_shift;      // # of bits to shift right by (for validity buffers)
+  size_type valid_count;
 };
 
+constexpr size_type copy_block_size = 512;
```

Comment:
This should be a template argument to `copy_buffer`, not a global.

Reply:
Will change as a followup PR if that's ok.  I'd like to avoid poking this code at this late hour if possible.